libbeat/docs/queueconfig.asciidoc

```diff
@@ -76,3 +77,147 @@ will be immediately available for consumption.
 
 The default values is 0s.
 
+[float]
+[[configuration-internal-queue-spool]]
+=== Configure the file spool queue
```

Comment:
Now that we have a couple of queue types, we should explain why users might choose to use a specific type of queue.

Reply:
Yeah. The interaction of a configured queue and the actual beat differs per beat. This would call  for some beat specific documentation. As this will require some more work, I'd prefer to postpone the per-beats documentation for now.